examples/window-app/telink/include/WindowCovering.h

```diff
@@ -0,0 +1,84 @@
+/*
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include "LEDWidget.h"
+#include "PWMDevice.h"
+
+#include <app/clusters/window-covering-server/window-covering-server.h>
+
+#include <cstdint>
+
+using namespace chip::app::Clusters::WindowCovering;
+
+class WindowCovering
+{
+public:
+    enum class MoveType : uint8_t
+    {
```

Comment:
// Window Covering Type
enum class WindowCoveringType : uint8_t
{
    Lift     = 0x00, // window coverings that lift up and down or slide left to right
    Tilt     = 0x01, // window coverings with vertical or horizontal strips
    Reserved = 0x02, // dont use
};

Cannot we reuse some of the delegation put in place within the server